plugin.script.omgtv/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.script.omgtv" name="OMGTV!!!" version="1.0.0" provider-name="OMGTV!!! LLC, New Jersey, US">
+    <requires>
+		<import addon="xbmc.python" version="3.0.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
```

Comment:
There is a different repo for plugins.  This needs to be submitted there for review.

https://github.com/xbmc/repo-plugins